.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: php
 
 php:
-  - 7.4
-  - 8.0
```

Comment:
Any reason to stop testing on PHP 8.0? It is still supported until October 2023.

I think for as long as we declare in composer.json to be compatible with PHP 7.4 we should also keep testing on 7.4. Probably we shouldn't drop PHP 7.4 support unless we plan to release a new major version.

I suppose the `./vendor/bin/drupal-check` test only works on PHP 8.1, but we can still run the remainder of the suite on older PHP versions.

Reply:
Drupal 10's requirement is to use 8.1. The CI may fail for 8.0 on D10

Reply:
PHP8 did not work

```
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - drupal/core-recommended[10.0.0-beta1, ..., 10.0.0-beta2] require symfony/process ~v6.1.3 -> satisfiable by symfony/process[v6.1.3].
    - symfony/process[v6.1.0-BETA1, ..., v6.1.3] require php >=8.1 -> your php version (8.0.23) does not satisfy that requirement.
    - Root composer.json requires drupal/core-recommended ^10@beta -> satisfiable by drupal/core-recommended[10.0.0-beta1, 10.0.0-beta2].
The command "composer install" failed and exited with 2 during .
```

https://app.travis-ci.com/github/jhedstrom/DrupalDriver/jobs/585619255

I've reverted my change so that we are only testing on PHP 8.1

If there is a different way to test with PHP 8.0 - please let me know and I will update this PR.